lib/Doctrine/Common/Annotations/Annotation/Target.php

```diff
@@ -54,7 +54,7 @@ final class Target
         'ANNOTATION' => self::TARGET_ANNOTATION,
     ];
 
-    /** @var array */
+    /** @var list<string> */
```

Comment:
I tried something more advanced here by using `list<'ALL'|'CLASS'|'METHOD'|'PROPERTY'|'ANNOTATION'>` but phpcs then said `Property \Doctrine\Common\Annotations\Annotation\Target::$value does not have @var annotation for its value.`. I could add it as a prefixed annotation, but I am torn: we use `psalm` annotations in other packages, but we don't use Psalm in this package, so maybe I should prefix them with `phpstan`? A `psalm` prefix would work for Phpstan, it recognizes it.

Reply:
Yeah, I'm torn by this in projects of mine too, but I guess we have to use the annotations of the tool we use.

Reply:
Ok, I will do a final pass to prefix those with `phpstan` when I'm done.

Reply:
@SenseException do you know in which case we should use a prefix, and in which case we shouldn't? `list`? `array<>`? `array{}`? All three? None of them?

Reply:
list and array{} at least IMHO

Reply:
Ok let's start with those 👍

Reply:
@ostrolucky addressed

Reply:
I use `array{}` only for specific structure, so I think your choices are fine.